Homework2/Task2TrainingManagementSystem/Training.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace Task2TrainingManagementSystem
+{
+    class Training : Entities, ICloneable
+    {
+        
+        public List<Entities> Trainingslist = new List<Entities>();
+        
+        public Training(string description) : base(description)
+        {
+            Description = description;
+        }
+
+        public void Add(Entities element)
+        {
+            this.Trainingslist.Add(element);
+        }
+
+        public static bool IsPractical(List<Entities> Testinglist)
```

Comment:
do not make it static and drop param, so the signature is:
`
  public static bool IsPractical()`